embulk-output-sqlserver/src/test/java/org/embulk/output/sqlserver/SQLServerOutputPluginTest.java

```diff
@@ -368,6 +369,33 @@ public void testNative() throws Exception
         assertTable(1, table);
     }
 
+    @Test
+    public void testJtds() throws Exception
+    {
+        boolean canTestJtds = false;
+        useJtdsDriver = true;
+        try {
+            new SQLServerOutputPluginTest().connect();
+            canTestJtds = true;
+
+            String table = "TEST1";
+            dropTable(table);
+            createTable(table);
+            insertRecord(table);
+            tester.run(convertYml("/sqlserver/yml/test-jtds.yml"));
+            assertTable(1, table);
+        } catch (Throwable t) {
```

Comment:
The test will succeed even if `tester.run` or `assertTable` throws an exception.
We should catch only exception thrown in `new SQLServerOutputPluginTest().connect()`.